strapi/strapi
# Chore: setting up nx releases

### What does it do?

- Add `@nx/js` packages
- Add release script and configs for nx releases

### Why is it needed?

- we are integrating nx releases and we need it for our auto releases workflow

### How to test it?

- every thing should build `yarn setup` should complete successfully
- if your run `yarn release --dry-run`  you should be able to test a dry run of a release
